engine/src/qlcfixturemode.cpp

```diff
@@ -389,6 +419,19 @@ bool QLCFixtureMode::loadXML(QXmlStreamReader &doc)
         }
     }
 
+    // Set acts on channels
+
+    foreach(ChannelActsOnData channelSctsOnData, listChannelsWithActsOnIndex) {
```

Comment:
Please respect the indentation of the rest if the code.
Parenthesis on a new line.
Space after conditionals (if, for, while, etc)